source/manual/howto-modify-change-note.html.md

```diff
@@ -76,4 +76,4 @@ The response to this is "false" but the update has taken place. You can confirm
 edition.save!(validate: false)
 ```
 
-Republish the document using a [Whitehall Rake task](https://docs.publishing.service.gov.uk/manual/republishing-content.html)
\ No newline at end of file
+Republish the document using a [Whitehall Rake task](https://docs.publishing.service.gov.uk/manual/republishing-content.html)
```

Comment:
Not sure what's changed here 🤔

Reply:
It's probably due to me and the last editor of this line having different line endings configured